src/styles/blocks/card.scss

```
@@ -0,0 +1,93 @@
+.card {
+  width: $card-width;
+  padding: 32px 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    display: block;
+    margin: 0 auto;
+
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin-top: 40px;
+
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-text-color;
+  }
+
+  &__code {
+    margin-top: 4px;
+
+    font-style: normal;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+
+    color: $second-text-color;
+  }
+
+  &__stars {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 16px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 24px;
+  }
+
+  &__price-title {
+    font-style: normal;
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: $second-text-color;
+  }
+
+  &__price-number {
+    font-style: normal;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $main-text-color;
+  }
+
+  &__btn {
+    margin-top: 16px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 40px;
+
+    background: $btn-color;
+    border: 1px solid;
+    border-radius: 5px;
+
+    font-style: normal;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-align: center;
+    text-transform: uppercase;
+    color: white;
+    transition: background-color 0.3s ease, color 0.3s ease;
+  }
+
+  &__btn:hover {
+    color: $btn-color;
+
+    background: white;
+    border-color: $btn-color;
```

Let's move this selector to `&__btn` selector
```suggestion
  &:hover {
    color: $btn-color;

    background: white;
    border-color: $btn-color;
```